samples/Serilog.Sinks.InfluxDB.Console.AppSettings/Program.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.CommandLine;
+using System.CommandLine.Builder;
+using System.CommandLine.Hosting;
+using System.CommandLine.Invocation;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using System.CommandLine.Parsing;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using System.Configuration;
+using Microsoft.Extensions.Configuration;
+using System.Linq;
+using System.Collections.Generic;
+using System.Net.Http;
+using Serilog.Debugging;
+using Serilog.Sinks.InfluxDB.Console.AppSettings;
+using System.IO;
+
+namespace Serilog.Sinks.InfluxDB.Console.AppSettings
+{
+    class Program
+    {
+        public static IConfiguration Configuration { get; } = new ConfigurationBuilder()
+            .SetBasePath(Directory.GetCurrentDirectory())
+            .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
+            .AddJsonFile($"appsettings.{Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") ?? "Production"}.json", optional: true)
+            .AddUserSecrets<Program>()
+            .AddEnvironmentVariables()
+            .Build();
+
+
+        static async Task Main(string[] args)
+        {
+            await BuildCommandLine()
+            .UseHost(_ => Host.CreateDefaultBuilder(),
+            host =>
+            {
+                SelfLog.Enable(System.Console.Out);
+                host.UseSerilog((hostingContext, loggerConfiguration) =>
+                {
+                    // could be configured here, but prefer earlier configuration
```

Comment:
Perhaps add a comment to explain config from file. We could add another sample with writing by code instead using Sink extensions methods ?